demo/index.html

```diff
@@ -88,6 +88,54 @@ <h1>Shadow DOM examples</h1>
           <auro-input disabled value="disabled" type="email" label="First name" helptext="Enter your first name" required id="disabledEmailTest01"></auro-input>
         </template>
       </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-ccWithIcon" format='{"creditCard": true, "icon": true}' icon="true" label="Credit Card Number with Icon" required></auro-input>
+        </template>
+      </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-cc" format='{"creditCard": true, "icon": true}' label="Credit Card Number without Icon" required></auro-input>
```

Comment:
This demo confuses me? For the format, `icon` is `true`, but this is the no-icon version?

Reply:
The key should have been removed when I switched course to use the property but was overlooked. Updated the PR.